server/policies/team.js

```diff
@@ -12,24 +11,7 @@ allow(User, "share", Team, (user, team) => {
   return team.sharing;
 });
 
-allow(User, "auditLog", Team, (user) => {
-  if (user.isAdmin) return true;
-  return false;
-});
-
-allow(User, "invite", Team, (user) => {
-  if (user.isAdmin) return true;
-  return false;
-});
-
-// ??? policy for creating new groups, I don't know how to do this other than on the team level
-allow(User, "group", Team, (user) => {
-  if (user.isAdmin) return true;
-  throw new AdminRequiredError();
-});
-
-allow(User, ["update", "export"], Team, (user, team) => {
+allow(User, ["update", "export", "manage"], Team, (user, team) => {
   if (!team || user.teamId !== team.id) return false;
-  if (user.isAdmin) return true;
-  throw new AdminRequiredError();
+  return user.isAdmin;
```

Comment:
Is there any specific reason we don't throw the `AdminRequiredError` here?

Reply:
Yea but it's hard to explain, "manage" is a special case in the authorization library.

If we throw here it the other team policies are not considered 😔

Reply:
I think It's indeed hard to explain 😅